doc/api/util.md

```diff
@@ -181,25 +181,25 @@ The `--throw-deprecation` command line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
-## util.format(format[, ...args])
+## util.format([format][, ...args])
 <!-- YAML
 added: v0.5.3
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The output string's formatting will again dependent on the
+                 type of the first argument. This reverts
+                 https://github.com/nodejs/node/pull/23162.
```

Comment:
I wouldn’t necessarily link to the PR, just mention that it reverts a former change (as we don’t display the `pr-url:` information, it’s just to make looking up things easier for us)